services/brig/fake-sqs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+function cleanup() {
+  printf 'cleaning up ...'
+  docker kill $(cat DOCKER_ID)
+  rm DOCKER_ID
+  printf done
+}
+
+trap cleanup EXIT
+
+docker run -d --cidfile DOCKER_ID -p 4568:4568 airdock/fake-sqs 
+while true
+do
+  curl --silent http://localhost:4568 -d "Action=CreateQueue&QueueName=integration-brig-events"
+  if [ $? -eq 0 ]; then
+    echo "Queue created!"
+    curl --silent http://localhost:4568 -d "Action=CreateQueue&QueueName=integration-brig-events-internal"
```

Comment:
let's consolidate this with the galley one, and move this script one level up to `services`, creating queues for both brig and galley. There'll be interference otherwise.

Reply:
Yup, was planning to do so, thanks!